frame/support/test/tests/final_keys.rs

```diff
@@ -201,27 +201,25 @@ pub type Block = generic::Block<Header, UncheckedExtrinsic>;
 
 frame_support::construct_runtime!(
 	pub enum Runtime
-	where
-		Block = Block,
-		NodeBlock = Block,
-		UncheckedExtrinsic = UncheckedExtrinsic,
+
 	{
-		System: frame_support_test,
+		System: frame_system,
 		FinalKeysNone: no_instance,
 		FinalKeysSome: instance,
 		Instance2FinalKeysSome: instance::<Instance2>,
 	}
 );
 
-impl frame_support_test::Config for Runtime {
```

Comment:
I guess this test was using a custom System pallet here? Not sure why though.

Reply:
Correct, which is not possible anymore.